src/components/fs/FS.cpp

```diff
@@ -0,0 +1,214 @@
+#include "FS.h"
+#include <cstring>
+#include <cstdlib>
+#include "drivers/SpiNorFlash.h"
+#include <littlefs/lfs.h>
+#include <lvgl/lvgl.h>
+
+using namespace Pinetime::Controllers;
+
+
+static constexpr size_t BLOCK_SIZE_BYTES = 4096;
+
+/*
+*   Interface between littlefs and SpiNorFlash
+*/
+
+namespace {
+    int read(const struct lfs_config *c, lfs_block_t block,
```

Comment:
The indentation in this file isn't consistent with it's header (or the rest of the project) at 4 spaces instead of 2.

Reply:
Thank you, I will review the recommendations.